stackslib/src/net/stackerdb/sync.rs

```diff
@@ -659,7 +659,11 @@ impl<NC: NeighborComms> StackerDBSync<NC> {
                     self.replicas.insert(naddr);
                 }
                 Err(_e) => {
-                    info!("Failed to begin session with {:?}: {:?}", &naddr, &_e);
+                    if naddr.addrbytes.is_in_private_range() {
+                        debug!("Failed to begin session with {:?}: {:?}", &naddr, &_e);
```

Comment:
Should this be test_debug! too?

Reply:
From my first look over it, I think both of them be `debug!` instead of `test_debug`. Not completely familiar with the previous conversations around it though.